ManatiCR/manatime
# 22: Projects listing

**Steps to test**
- See if views has been added successfully.
- Create projects nodes to test.

**DoD**
- As an administrator I want to go /projects and get a table with a list of Project Name, Client Names.
- In that view, I need to have direct links to edit projects.

[issue link](https://github.com/ManatiCR/manatime/issues/22)
